source/enemies/elegantstella.js

```diff
@@ -57,13 +57,13 @@ module.exports = new EnemyTemplate("Elegant Stella",
 	description: `Deal minor ${getEmoji("Light")} damage and slowing @e{Misfortune} to all foes`,
 	priority: 0,
 	effect: (targets, user, adventure) => {
-		const pendingMisfortune = 1;
+		let pendingMisfortune = 1;
 		if (user.crit) {
 			pendingMisfortune += 7;
 		}
 		const { resultLines, survivors } = dealDamage(targets, user, user.getPower() + 25, false, "Light", adventure);
 		changeStagger(survivors, user, ESSENCE_MATCH_STAGGER_FOE);
-		return resultLines.concat(generateModifierResultLines(addModifier(survivors, pendingMisfortune)));
+		return resultLines.concat(generateModifierResultLines(addModifier(survivors, { name: "Misfortune", stacks: pendingMisfortune })));
```

Comment:
This line would actually not be broken with the pattern of fix shown above, but good catch when going with this style.
```suggestion
		return resultLines.concat(generateModifierResultLines(addModifier(survivors, pendingMisfortune)));
```